app/src/main/java/com/kickstarter/ui/fragments/CancelPledgeFragment.kt

```diff
@@ -90,8 +95,15 @@ class CancelPledgeFragment : BaseFragment<CancelPledgeViewModel.ViewModel>() {
         val ksString = (activity?.applicationContext as KSApplication).component().environment().ksString()
         val amount = amountAndProjectName.first
         val name = amountAndProjectName.second
-        cancel_prompt.text = ksString.format(getString(R.string.Are_you_sure_you_wish_to_cancel_your_amount_pledge_to_project_name),
+        val formattedString = ksString.format(getString(R.string.Are_you_sure_you_wish_to_cancel_your_amount_pledge_to_project_name),
                 "amount", amount, "project_name", name)
+
+        val spannableString = SpannableString(formattedString)
```

Comment:
Ah ok so at this point formatted string already has the data interpolated, and using `SpannableString` is really just to add bold-ness to the string for the amount and name of the project, right?

I'm getting it!

Reply:
...I think

Reply:
yes!